app/views/layouts/application.html.haml

```diff
@@ -5,19 +5,21 @@
 !!!
 %html{lang: I18n.locale.to_s, dir: (rtl? ? "rtl" : "ltr")}
   %head{prefix: og_prefix}
-    = render "head"
     %meta{name: "viewport", content: "width=device-width, initial-scale=1"}/
 
-    = include_color_theme
-    = old_browser_js_support
+    - content_for :javascript do
+      = old_browser_js_support
+
+      <!--[if IE]>
+      = javascript_include_tag :ie
+      <![endif]-->
 
-    <!--[if IE]>
-    = javascript_include_tag :ie
-    <![endif]-->
+      = javascript_include_tag :main, :templates
+      = load_javascript_locales
+      = translation_missing_warnings
```

Comment:
This line adds some CSS (https://github.com/diaspora/diaspora/blob/365441ede260212c406bc5ee4bde2764c1ff5e36/app/helpers/layout_helper.rb#L39), so I wouldn't add this to `content_for :javascript` but move it next to the color themes.